openproblems/tasks/spatial_decomposition/datasets/pancreas.py

```diff
@@ -0,0 +1,19 @@
+from ....data.pancreas import load_pancreas
+from ....tools.decorators import dataset
+from ._utils import generate_synthetic_dataset
+
+
+@dataset("Pancreas (average)")
+def pancreas_average(test=False):
```

Comment:
You need to make sure these datasets are properly imported in the `datasets/__init__.py`

Reply:
now they are, this was bad sorry